stores/productmanage.ts

```
@@ -0,0 +1,92 @@
+        })
+      })
+      return arr.sort((a, b) => a.sort - b.sort)
+    },
+  },
+  actions: {
+    // 货品列表
+    async getProductList(pamars: ProductReq) {
+      try {
+        const { data } = await https.post<ProductRes, ProductReq>('/product/list', pamars)
+        if (data.value.code === HttpCode.SUCCESS) {
+          pamars.page === 1 ? this.productList = data.value.data.list : this.productList = this.productList.concat(data.value.data.list)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)